storybookjs/storybook
# React: Add addon-onboarding as part of init

closes https://github.com/storybookjs/storybook/issues/22872, closes https://github.com/storybookjs/storybook/issues/22870



## What I did

This PR adds @storybook/addon-onboarding as part of the initiate step and execute dev with `--initial-path=/onboarding` to the CLI of (for now) React projects

This should not be merged until the addon-onboarding is ready.

## How to test

1. Build all packages
2. Create a new project with the CLI e.g. `./code/lib/cli/bin/index.js init`
3. Expect Storybook to open with the onboarding flow

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
